verticapy/pipeline/connection.yaml

```diff
@@ -0,0 +1,5 @@
+host: "127.0.0.1"
```

Comment:
what is it used for?

Reply:
We need the connection.yaml to specify where the user would like the pipeline to be built. It makes it easier if they want to rerun the pipeline on multiple servers. It's currently filled in with how to connect to VerticaDB if a user is using VerticaPyLab.